source/physics/elasticity/standard_tensors.inst.in

```diff
@@ -0,0 +1,41 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2016 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+
+for (deal_II_dimension : DIMENSIONS)
+{
+    template
+    class StandardTensors<deal_II_dimension>;
```

Comment:
Oh, ok we're on the same page now. I think that you might have skipped over their definitions, which are now [here](https://github.com/dealii/dealii/pull/3034/files#diff-48b3ed19fabc1d678153b1734437b782R22). So is this ok?